pgcli/pgclirc

```diff
@@ -143,6 +143,9 @@ enable_pager = True
 # Use keyring to automatically save and load password in a secure manner 
 keyring = True
 
+# When the <Tab> key is pressed on an empty line, use 4 spaces instead of \t
+expand_tab = False
```

Comment:
Can we pick one and make it a default? I really hate adding new config options. 

Given that pgcli is a REPL, I honestly don't think users will care if we choose tab or spaces.